pyproject.toml

```diff
@@ -2,5 +2,6 @@
 requires = [
     "setuptools == 60.5", "cython == 0.29.26", "wheel == 0.37",
     "numpy==1.20; python_version=='3.7'",
-    "numpy==1.22; python_version>='3.8'",
+    "numpy==1.22; python_version>='3.8',<'3.11'",
```

Comment:
@jwodder -- is this the correct specification for a range of python versions here?

Reply:
pip gave me the answer - No ;)
```
× Can not process file:///home/yoh/proj/misc/pylibjpeg-libjpeg
╰─> This package has an invalid `build-system.requires` key in pyproject.toml.
    It contains an invalid requirement: "numpy==1.22; python_version>='3.8',<'3.11'"

note: This is an issue with the package mentioned above, not pip.
hint: See PEP 518 for the detailed specification.
```

Reply:
nevermind, figured it out, fix is coming